ironic/drivers/modules/redfish/boot.py

```diff
@@ -407,18 +407,34 @@ def _validate_instance_info(self, task):
         d_info = _parse_deploy_info(node)
         deploy_utils.validate_image_properties(task, d_info)
 
-    def _validate_vendor(self, task):
+    def _validate_vendor(self, task, managers):
+        """Validates vendor specific requirements for the task's node.
+
+        :param task: a TaskManager instance containing the node to act on.
+        :param managers: Redfish managers for Redfish system associated
+            with node.
+        :raises: InvalidParameterValue if vendor not supported
+        """
         vendor = task.node.properties.get('vendor')
         if not vendor:
             return
 
         if 'Dell' in vendor.split():
+            # Check if iDRAC fw >= 6.00.00.00 that supports virtual media boot
+            bmc_manager = [m for m in managers
+                           if m.manager_type == sushy.MANAGER_TYPE_BMC]
+            if bmc_manager:
+                fwv = bmc_manager[0].firmware_version.split('.')
+                if int(fwv[0]) >= 6:
```

Comment:
The constraint is a bit too strict for our use-case, see [here](https://github.com/sapcc/ironic/pull/21/files#diff-2f42c41625f1622b9073e25f7fbf087d5ebbd98e258fdfc1a43f6d0777a3ea94R182-R187).
It is the save value for both AMD and Intel. Not sure, though, if we need to change it, considering we are
- not using virtual media boot yet
- are upgrading the BMCs to something more current anyway
-

Reply:
I've checked we want to upgrade all idracs to a 6.x version soon anyway because with version 6.x the redfish syntax changes and this makes it also easier for our other automation to have a stable version for all nodes.